tooling/image-sync/Dockerfile

```diff
@@ -5,7 +5,7 @@ ADD . .
 # https://github.com/microsoft/go/tree/microsoft/main/eng/doc/fips#build-option-to-require-fips-mode
 ENV CGO_ENABLED=1 GOFLAGS='-tags=requirefips'
 
-RUN make image-sync
+RUN go build -tags=containers_image_openpgp,requirefips .
```

Comment:
Ah I did this because the make target doesn't work from the Dockerfile's perspective due to line 4
```
include ../../dev-infrastructure/configurations/$(CONFIG_PROFILE).mk
```